### PR TITLE
Removing the "Zeit" term from Vercel deployment

### PR DESCRIPTION
**Pull request by andy-mishechkin**
# Description
Improvement. 
"Zeit" term has been removed from all Vercel deployment scripts
#### Related issue:
https://github.com/actions/virtual-environments/issues/882

## Check list
- [X] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/vercel.sh [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 if ! command -v now; then
-    echo "Now link to Zeit Vercel CLI was not created"
+    echo "[Now] symlink to Vercel CLI was not created"
     exit 1
 fi
 
 # Document the installed version
 echo "Document the installed version"
-DocumentInstalledItem "Zeit Vercel CLI ($(vercel --version))"
+DocumentInstalledItem "Vercel CLI ($(vercel --version))"
```
Comment (AlenaSviridenko) [inline images/linux/scripts/installers/vercel.sh:31]: I believe tool should be documented according to this guide: https://github.com/actions/virtual-environments/blob/master/CONTRIBUTING.md#submitting-a-pull-request

> add changes to document tool name and version to the software report generator: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 that is used to generate software README file, e.g. Windows2019-Readme.md

Comment (miketimofeev) [reply images/linux/scripts/installers/vercel.sh:31]: It's for windows only at the moment.

Comment (vmapetr) [reply images/linux/scripts/installers/vercel.sh:31]: There is no separate software report generator on linux images, and it is a common practice that we install, validate, and create documentation in the same sh script.

Comment (AlenaSviridenko) [reply images/linux/scripts/installers/vercel.sh:31]: oh, that's linux 🤦‍♀️. sorry for the confusion

Comment (andy-mishechkin) [reply images/linux/scripts/installers/vercel.sh:31]: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 using in Windows deployments only

File: images/linux/scripts/installers/vercel.sh [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-# Install the Zeit Vercel CLI
+# Install the Vercel CLI
 npm i -g vercel
 
 # Validate the installation
 echo "Validate the installation"
```
Comment (vmapetr) [inline images/linux/scripts/installers/vercel.sh:14]: Perhaps, it's make sense to move this block below symlink creation, to separate installation from validation.

Comment (andy-mishechkin) [reply images/linux/scripts/installers/vercel.sh:14]: IMHO In this case, if Vercel installation will be failed we can get additional error in symlink creation.
I think, better to check Vercel installation before symlink creation as it already done

**Pull request merged**